main.go

```diff
@@ -202,22 +225,34 @@ func buildParseErrorString(target, e string, err error) string {
 	return msg
 }
 
+func SetupDeferredAccessLogging(accessLogger *zap.Logger, accessLogDetails *carbonapipb.AccessLogDetails, t time.Time, logAsError bool)  {
+	accessLogDetails.Runtime = time.Since(t).String()
+	if logAsError {
+		accessLogger.Error("Request failed", zap.Any("data", *accessLogDetails))
+	} else {
+		accessLogDetails.HttpCode = http.StatusOK
+		accessLogger.Info("Request served", zap.Any("data", *accessLogDetails))
+	}
+}
+
 func renderHandler(w http.ResponseWriter, r *http.Request) {
 	t0 := time.Now()
 	uuid := uuid.NewV4()
+
 	// TODO: Migrate to context.WithTimeout
 	// ctx, _ := context.WithTimeout(context.TODO(), config.ZipperTimeout)
 	ctx := util.SetUUID(r.Context(), uuid.String())
 	username, _, _ := r.BasicAuth()
 
-	logger := zapwriter.Logger("render").With(
+	logger := zapwriter.Logger("render")
+	logger.With(
 		zap.String("carbonapi_uuid", uuid.String()),
 		zap.String("username", username),
 	)
 
 	srcIP, srcPort := splitRemoteAddr(r.RemoteAddr)
 
-	accessLogger := zapwriter.Logger("access")
+	accessLogger := logger
```

Comment:
accessLogger should be a different logger.
Purpose for that is that we can configure per-loger output (verbosity, format, output to file or whatever), if it'll reuse same 'logger' as global one - we won't be able to do that anymore.